GETTING-STARTED.MD

```diff
@@ -0,0 +1,25 @@
+# Adding the Extension
+To get started with the Vuln Cost Security Scanner extension for Visual Studio Code (VSCode), you must first install the *Vuln Cost - Security Scanner* by `snyk-security.vscode-vuln-cost` from the **Extensions Marketplace**. You may need to restart VSCode to see warnings.
```

Comment:
Great call out! In some cases, people restarted VSCode multiple times before it worked. (Currently we don't know why) If there are any problems make sure VS Code is fully shut down and restarted.